packages/client/src/AccountsClient.js

```diff
@@ -264,6 +270,7 @@ export class AccountsClient {
   }
 
   async requestPasswordReset(email?: string): Promise<void> {
```

Comment:
This is how it was, but come to think of it, changing it basically means nothing :)
Although I would like the email to be validated before making a network call.